app/models/playlist.rb

```diff
@@ -5,12 +5,13 @@ class Playlist < ActiveRecord::Base
 
   scope :albums,           -> { where(is_mix: false).where(is_favorite: false) }
   scope :favorites,        -> { where(is_favorite: true) }
-  scope :for_home,         -> { select('distinct playlists.user_id, playlists.*').recent.only_public.with_preloads }
+  scope :for_home,         -> { select('distinct playlists.user_id, playlists.*').recently_published.only_public.with_preloads }
```

Comment:
I forgot about this too, but we probably need to change `playlists_controller#all` to have the same logic (sorting by published vs. recent). I believe then we can kill the `recent` scope below.